tests/sentry/search/events/test_builder.py

```diff
@@ -565,3 +568,134 @@ def test_disable_auto_aggregation(self):
         # With count_unique only in a condition and no auto_aggregations this should raise a invalid search query
         with self.assertRaises(InvalidSearchQuery):
             query.get_snql_query()
+
+
+def _metric_percentile_definition(quantile, field="transaction.duration") -> Function:
+    return Function(
+        "arrayElement",
+        [
+            Function(
+                f"quantilesMergeIf(0.{quantile.rstrip('0')})",
```

Comment:
Hmm this was cause of this error I got when trying to use `quantile`:
```
Received exception from server (version 20.3.9):
Code: 43. DB::Exception: Received from localhost:9000. DB::Exception: Illegal type AggregateFunction(quantiles(0.5, 0.75, 0.9, 0.95, 0.99), Float64) of argument for aggregate function with Merge suffix, because it corresponds to different aggregate function: quantiles instead of quantile.
```

and looking at the `describe metrics_distributions_local` I see the `type` for the `percentiles` column is
```AggregateFunction(quantiles(0.5, 0.75, 0.9, 0.95, 0.99), Float64)```

Reply:
Oh that's interesting. OK I didn't realize it worked that way. Carry on.